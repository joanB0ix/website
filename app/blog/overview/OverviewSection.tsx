import { Section } from "@/components/Section";

export default function OverviewSection() {
  return (
    <Section title="My Blog">
      <p>This is my blog, where I write about things that interest me.</p>
    </Section>
  );
}
