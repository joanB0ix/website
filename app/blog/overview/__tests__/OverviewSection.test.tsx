import { render, screen } from "@testing-library/react";
import OverviewSection from "../OverviewSection";

describe("OverviewSection", () => {
  it("Should render correctly", () => {
    render(<OverviewSection />);

    const sectionTitle = screen.getByText("My Blog");
    expect(sectionTitle).toBeInTheDocument();
  });

  it("Should match snapshot", () => {
    const { asFragment } = render(<OverviewSection />);
    expect(asFragment()).toMatchSnapshot();
  });
});
