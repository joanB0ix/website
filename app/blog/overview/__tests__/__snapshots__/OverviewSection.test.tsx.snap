// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OverviewSection Should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="my-6"
  >
    <h2
      class="font-bold pb-2"
    >
      My Blog
    </h2>
    <p>
      This is my blog, where I write about things that interest me.
    </p>
  </div>
</DocumentFragment>
`;
